blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+```
+node_1,node_2
+0,1794
+0,3102
+0,16645
+```
+Dataset consists of Facebook pages (from November 2017). It represents a network of verified Facebook pages of different categories. Nodes represent the pages and relationships are mutual likes among them. The nodes are reindexed (starting from 0) to achieve a certain level of anonymity. Since **Memgraph** imports query faster when it has indexes, we will create one for all nodes with the label `Page` on the `id` property.  
+
+
+```python
+memgraph.execute(
+    """
+    CREATE INDEX ON :Page(id);
+    """
+)
+```
+
+Now we will make *full* usage of our "hack" from before. We will list through our local files in the `./data/facebook_clean_data/` folder and create the path from them concatenating our file names to the `/samples/` folder which will then represent the path to **CSV** files in *Docker* container.
+
+
+
+```python
+import os
+from os import listdir
+from os.path import isfile, join
+csv_dir_path = os.path.abspath("./data/facebook_clean_data/")
+csv_files = [f"/samples/{f}" for f in listdir(csv_dir_path) if isfile(join(csv_dir_path, f))]
+
+```
+
+Once we have all the `CSV` files, we will load all of them with the following query:
+
+```python
+for csv_file_path in csv_files:
+    memgraph.execute(
+        f"""
+        LOAD CSV FROM "{csv_file_path}" WITH HEADER AS row
+        MERGE (p1:Page {{id: row.node_1}}) 
+        MERGE (p2:Page {{id: row.node_2}}) 
+        MERGE (p1)-[:LIKES]->(p2);
+        """
+    )
+
+```
+
+You can find out more about the `LOAD CSV` clause importing CSV files in our [docs](https://memgraph.com/docs/memgraph/import-data/load-csv-clause).
+
+We are all set to use PageRank and Louvain community detection algorithms with Python to find out which pages in our network are most important and to find all the communities we have in a network.
+
+## PageRank importance analysis
+Now, we will execute PageRank to find important pages of a Facebook dataset. To read more about how does **Pagerank** works, you can go to our **[docs](https://memgraph.com/docs/mage/query-modules/cpp/pagerank)** page. All algorithms mentioned in [introduction](#introduction) part are developed by **[Nvidia](https://rapids.ai/)** and now are integrated under **MAGE - Memgraph Advanced Graph Extensions** . Our goal in **Memgraph** is for you to have it very easy when it comes to using algorithms on graph databases and getting results fast. They are implemented in C++ or Python. There is a lot of other graph analytics available for you under **MAGE**, from **graph neural networks** all to different centralities. You can learn more about them [here](https://memgraph.com/docs/mage/algorithms) and we have a series of tutorials you can read to learn how to use such analytics to classify nodes, predict relationships, and more. Check out our [tutorials](https://memgraph.com/categories/tutorials) for more examples.
```

Comment:
Now, let's execute PageRank... 
All algorithms mentioned in the introduction are (OR WERE?) developed by Nvidia, and they are now integrated within MAGE - bal bla bla bla.

Our goal in Memgraph is to make it easy for you to use algorithms on graph databases and get results quickly. Graph algorithms in MAGE are implemented in C++ or Python.

MAGE also includes many other graph analytics algorithms, from **graph neural networks** to many different centralities. You can learn more about them on the MAGE docs page, and we've also written many tutorials on how to use such analytics to to classify nodes, predict relationships, and more, so do not hesitate to check out our tutorials as well.